### ISSUE TITLE
install faild: npm ERR! code EUNSUPPORTEDPROTOCOL

### ISSUE THREAD

**Issue by jingyuexing** [jd-solanki/anu#32]
```
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "link:": link:/Users/jd/Projects/oss/anu/packages/anu-vue/dist
```
@jd-solanki

**Comment by jd-solanki** [comment]
Do you have pnpm?

**Comment by jingyuexing** [comment]
> Do you have pnpm?

Emmm,not yet

**Comment by jd-solanki** [comment]
Are you running Anu in dev mode (forking & contributing) or using it as lib for your project?

**Comment by jd-solanki** [comment]
Which package manager you are using?

**Comment by jingyuexing** [comment]
> Which package manager you are using?

npm

**Comment by jingyuexing** [comment]
> Can you try using pnpm?

I have tried it, the network is not good

**Comment by jingyuexing** [comment]
So, I went back to npm

**Comment by jd-solanki** [comment]
I have released [v0.9.0](https://github.com/jd-solanki/anu/releases/tag/v0.9.0)

Can you please check if now with fresh installation?

Awaiting your response.
